lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
Hi @wtrocki, 

Sorry, if I was not clear enough in the last messages. I removed the last two comments and create this one where the idea is more clear about: Why I just don't change the code to follow your suggestion for we move forward here?

- I understand that your suggestion doesn't will be attended situations as following, we need to improve it a little more to make it more flexible. 

`````
 if (!params.json) {
        if (data && data.list && data.list.length > 0) {
          params._table = common.createTableForNotifications(data.list);
          return cb(undefined, params);
        } else {
          return cb(util.format(i18n._("Not found notifications for the app '%s' into the env '%s'."), params.app, params.env));
        }
      } else {
        return cb(undefined, data.list || data);
      }
````
- Remove the _table will broke a lot of tests that we have in the project. 
- I think we should first solve the main technical debts which are making all commands follow the same standard and be used all in the same way for the case when the project is used as a dependency. After this main changes, we will have more visibility of the code and we will be in a better position to make improvements as that. 
- We already have a JIRA for after we solve the main technical debts as describe above we improve and make the --json parameter as a global thing and we can at this moment find the best approach to improve the output related this parameter either.  
- The test team is working on automated a lot of things via fhc tool and it is a simple thing that is making a big noise and unnecessary effort. 
- Since you already test and check that it worked fine, and as you said in or comment "Note that this is only suggestion", could we agree to move forward with it in a simple way and after improve as you suggested which is already planned in another JIRA [FH-3650](https://issues.jboss.org/browse/FH-3650)